var/spack/repos/builtin/packages/nim/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Nim(Package):
+    """Nim is a statically typed compiled systems programming language.
+    It combines successful concepts from mature languages like Python,
+    Ada and Modula."""
+
+    homepage = "https://nim-lang.org/"
+    url = "https://nim-lang.org/download/nim-0.20.0.tar.xz"
```

Comment:
If you use the github URL then you can use a consistent url of all versions: https://github.com/nim-lang/Nim/archive/v0.20.0.tar.gz